### PR TITLE
New package: MicrosoftCorporation.MicrosoftOffice version 16.0.18129.20158

### PR DESCRIPTION
**Pull request by powellalexander**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199513)

### CODE REVIEW

**Comment by powellalexander** [comment]
@microsoft-github-policy-service agree

File: manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftOffice
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.Office
```
Please update file name and path to match this change.

File: manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftOffice
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.Office
```
Please update file name and path to match this change.

File: manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftOffice
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.Office
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @powellalexander,

This package already exists, as `Microsoft.Office`.

**Comment by stephengillie** [comment]
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\m\MicrosoftCorporation\MicrosoftOffice\16.0.18129.20158', expected 
'manifests\m\Microsoft\Office\16.0.18129.20158'

The file names and paths haven't been updated yet. They have to match.

**Pull request closed without merging**